features/push/push-init.jsonnet

```diff
@@ -106,5 +106,20 @@ local tags = import '../tags.libsonnet';
     label: 'Registering device token',
     tag: tags.pushTag,
     log: 'Registering device token: {{token}}',
+    next: 'automatic-profile-push-token-registration'
+  },
+
+  {
+    id: 'automatic-profile-push-token-registration',
+    label: 'Automatically registering token to profile',
+    tag: tags.pushTag,
+    log: 'Automatically registering device token: {{token}} to newly identified profile: {{userId}}',
```

Comment:
its not going to be newly everytime no? on app startup when we get token and register it again, we will use the same profile?